src/test/java/org/spectra/cluster/model/consensus/GreedyConsensusSpectrumTest.java

```diff
@@ -1,19 +1,23 @@
 package org.spectra.cluster.model.consensus;
 
+import lombok.extern.slf4j.Slf4j;
 import org.junit.Assert;
 import org.junit.Test;
+import org.spectra.cluster.filter.binaryspectrum.HighestPeakPerBinFunction;
 import org.spectra.cluster.io.MzSpectraReader;
 import org.spectra.cluster.model.cluster.GreedySpectralClusterTest;
 import org.spectra.cluster.model.spectra.BinaryPeak;
 import org.spectra.cluster.model.spectra.IBinarySpectrum;
+import org.spectra.cluster.normalizer.BasicIntegerNormalizer;
+import org.spectra.cluster.normalizer.CumulativeIntensityNormalizer;
+import org.spectra.cluster.normalizer.SequestBinner;
 import org.spectra.cluster.similarity.CombinedFisherIntensityTest;
 import org.spectra.cluster.similarity.IBinarySpectrumSimilarity;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
 
+@Slf4j
```

Comment:
We need to update this test so it catches cases were the consensus spectrum is dirty and "getMzVector" or any of the IBinarySpectrum associated functions are called. To make sure that there are no discrepant states.

Reply:
**DONE**